sphinx/source/docs/dev_guide/testing.rst

```diff
@@ -12,11 +12,11 @@ be viewed at: https://travis-ci.org/bokeh/bokeh.
 
 To run the just the python unit tests, run the command::
 
-    $ python -c "import bokeh; bokeh.test()"
+    python -c "import bokeh; bokeh.test()"
```

Comment:
Err, just noticed that this is a shell script executing python, it should be `.. code-block:: python`